modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -701,6 +701,26 @@ namespace cv {
 namespace gimpl {
 namespace onnx {
 
+static GraphOptimizationLevel convertToGraphOptimizationLevel(const int opt_level) {
+    switch (opt_level) {
+    case ORT_DISABLE_ALL:
```

Comment:
Btw, what is the purpose of such checks? 
```
case ENUM_VALUE:
    return ENUM_VALUE
```

Can they be simplified?

Reply:
Most checks are in that form:
    case ORT_DISABLE_ALL:
        return ORT_DISABLE_ALL;
    case ORT_ENABLE_BASIC:
        return ORT_ENABLE_BASIC;
    case ORT_ENABLE_EXTENDED:
        return ORT_ENABLE_EXTENDED;
    case ORT_ENABLE_ALL:
        return ORT_ENABLE_ALL;

The only exception is for values over 99 where there are all passed as 99 (ORT_ENABLE_ALL). This part is manly copy paste from ONNXRuntime but adapted for this project

Reply:
Used static_cast to perform the transformation. Error handling is being done by onnx runtime